styles.css

```diff
@@ -0,0 +1,1115 @@
+  background-color: #2230d2;
+  color: #fff;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: #fff;
+  cursor: pointer;
+}
+
+button:disabled {
+  color: #5e6c84;
+  background-color: #e5e5e5;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.btn-two {
+  border: 1px solid #4053fc;
+  padding: 7px;
+  border-radius: 4%;
+  font-size: 17px;
+  font-weight: 300;
+  text-transform: capitalize;
+}
+
+a .btn-two {
+  color: #fff;
+  text-decoration: none;
+}
+
+.languages {
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+}
+
+.section-about {
+  background-color: white;
+  border-top-right-radius: 100px;
+}
+
+.section-about h1 {
+  position: static;
+  width: 50px;
+  height: 52px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 35px;
+  letter-spacing: 0.37px;
+  color: #091e42;
+  margin: 40px 10px 30px 0;
+  font-size: 35px;
+}
+
+.about-grid {
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+}
+
+.about-grid > article {
+  padding-bottom: 50px;
+}
+
+.lang-font {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  color: #344563;
+}
+
+.lang {
+  margin: 30px;
+  margin-top: 50px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang-margin {
+  margin-top: 4px;
+}
+
+.lang-2 {
+  margin: 20px;
+  padding-top: 6px;
+  padding-bottom: 25px;
+  padding-right: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang3 {
+  border-bottom: 1px solid rgb(202, 200, 200);
+}
+
+.icons {
+  height: 15px;
+  width: 15px;
+}
+
+.language {
+  display: flex;
+  margin-left: 10px;
+  margin-bottom: 10px;
+  background-color: #f6f6f9;
+  border-radius: 8px;
+  padding: 10px;
+}
+
+.lang-text {
+  font-family: Poppins, sans-serif;
+  color: #344563;
+  padding-top: 10px;
+  padding-left: 15px;
+  font-style: normal;
+}
+
+.contact-section {
+  background: #fff;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  background: #6070ff;
+  background-image: url(./images/contact-shapesm.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  border: 1px solid white;
+  border-top-left-radius: 110px;
+  padding-bottom: 48px;
+}
+
+.con-title {
+  color: #fcfcfc;
+  justify-content: center;
+  font-weight: bold;
+  line-height: 40px;
+  font-size: 40px;
+  font-family: Poppins, sans-serif;
+  margin: 150px auto 12px;
+}
+
+.con-paragraph {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  justify-content: center;
+  text-align: center;
+  font-weight: normal;
+  margin: 0 auto;
+  padding: 10px 50px;
+}
+
+.con-form {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  gap: 20px;
+}
+
+.form-text {
+  border: 1px solid white;
+  border-radius: 8px;
+  height: 48px;
+  padding: 15px;
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] Great job implementing button interactions, kindly reduce the border-radius of the input elements to match the design.

### Your website
![Screen Shot 2022-09-03 at 10 02 11 AM](https://user-images.githubusercontent.com/59416136/188259961-f62eec4a-5883-4bf3-8897-11e2dae8f53b.png)

### Figma's design
![Screen Shot 2022-09-03 at 10 04 27 AM](https://user-images.githubusercontent.com/59416136/188259996-ffc48928-06a7-4299-a672-dabbd80bcc35.png)

Reply:
thanks I have made the corrections